### DISCUSSION TITLE
Oredict

### DISCUSSION THREAD

**Discussion by TheGeneralJR1** [SirSquidly/Oceanic-Expanse#35] [Ideas]
This is a small addition idea for the mod, but perhaps you could oredict coconuts or at least cracked coconuts as cropCoconut. Whatever Pam's Harvestcraft uses. And maybe oredict any other item that may be similarly used in other mods. Because I prefer your coconuts. Also, maybe you could add some kind of gun perhaps to fire said coconuts. Maybe it fires in spurts. Jk.

**Comment by Sethgno** [comment]
i would also like a coconut gun that fires in spurts. please make it so that when it shoots you, its gonna hurt.

**Comment by SirSquidly** [comment]
Oh yeah, adding Ore Dictionary for Coconuts is a clear oversight on my part, I'll slip that in the following update. I'll add the Cracked Coconut to `cropCoconut` and `foodCoconutmilk` so it can be used for all of the Pam's Cooking and anything using the same entries.

Also, a Coconut Gun is totally going on the future feature list.

**Reply by SirSquidly** [reply]
Follow up, I've added the Ore Dictionary changes in version 1.0.7.
